test/JDBC/expected/alter_authorization_change_db_owner-vu-verify.out

```diff
@@ -390,12 +390,11 @@ use master
 go
 
 -- cannot rename the database without sysadmin role, even when you're the owner already
--- ALTER DATABASE is not yet supported in Babelfish:
 alter database change_owner_db modify name = change_owner_db_renamed
 go
 ~~ERROR (Code: 33557097)~~
 
-~~ERROR (Message: 'ALTER DATABASE' is not currently supported in Babelfish)~~
+~~ERROR (Message: User does not have permission to rename the database 'change_owner_db', the database does not exist, or the database is not in a state that allows access checks.)~~
```

Comment:
Since with this change, we not blocking the whole `ALTER DATABASE` anymore. We are only allowing rename db, Please add testcase for other statement variants to validate that they are still throwing the unsupported error.

Reply:
```
-- simple unsupported query which backend parser can't understand. should throw an error with nice error message
ALTER DATABASE blah SET ANSI_PADDING OFF;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: 'ALTER DATABASE' is not currently supported in Babelfish)~~
```
There is already tests for this!